README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+
+### Brief Biography Arline  
+My name is Arline Lee. I am a museum science teacher. I am unabashedly and unironically a huge fan of: 
+* **Science Fiction**  
```

Comment:
Nice nested list. Note you can use the same symbol for each level, unless you want to distinguish between bullet points, numbers, letters, etc.

```
* **Science Fiction**  
   * I have recently come around to some genre permeability with fantasy
      * If anyone is interested N.K. Jemisin's *The Fifth Seasson* is a perfect example of "hard sci-fi" fantasy. 
  * My favorite series is *Deep Space Nine*, but Captain Janeway is my favorite captain 
```

* **Science Fiction**  
   * I have recently come around to some genre permeability with fantasy
      * If anyone is interested N.K. Jemisin's *The Fifth Seasson* is a perfect example of "hard sci-fi" fantasy. 
  * My favorite series is *Deep Space Nine*, but Captain Janeway is my favorite captain